packages/11ty/_lib/chalk/index.js

```diff
@@ -41,7 +41,8 @@ module.exports = function (prefix='', loglevel=2) {
   const logFn = (type) => {
     const log = logger[type]
     const style = styles[type]
-    return (message) => log(style(chalk.bold(`[${prefix}]`), message))
+    const prefixSpacer = prefix.length > 30 ? " " : " ".repeat(30 - prefix.length)
+    return (message) => log(style(chalk.bold(`[quire] ${type.toUpperCase()}\t`), `${prefix}${prefixSpacer}`, message))
```

Comment:
@geealbers it may work nicely to use the `String.prototype.padEnd()` method, which accepts a target length and the padding character. 
```suggestion
    prefix = prefix.padEnd(30, '\u0020')
    return (message) => log(style(chalk.bold(`[quire] ${type.toUpperCase()}\t`), `${prefix}`, message))
```

Reply:
Ah yes, that's nicer, and works in testing. Thanks!